roles/telemetry_autoscaling/tasks/test_autoscaling.yml

```diff
@@ -57,7 +57,7 @@
   register: busy_process
   with_items: "{{ vnf_instance_ip.stdout_lines }}"
 
-- name: RHOSO-12660 Verify that the `cpu_alarm_high` alarm has been triggered
+- name: [TEST] Verify alarm has been triggered
```

Comment:
I would say its better to add the alarm name, to verify which alarm we are talking about.